osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -359,6 +359,15 @@ protected override void Update()
             base.Update();
 
             detailsArea.Height = wedgesContainer.DrawHeight - titleWedge.LayoutSize.Y - 4;
+
+            float widescreenBonusWidth = Math.Max(0, DrawWidth / DrawHeight - 2);
+
+            mainGridContainer.ColumnDimensions = new[]
```

Comment:
My one concern about this would be performance because the setter for `ColumnDimensions` is a bit not smart:

https://github.com/ppy/osu-framework/blob/f049aef38bd2cc6f849c5068b61b4aadcf729460/osu.Framework/Graphics/Containers/GridContainer.cs#L104-L117

Like that `==` guard there isn't doing much at all is it. That said I can't see it having a meaningful impact on invalidation count so...

Reply:
I did check invalidations before PRing and agree that nothing looked too bad. 🤷